pyworklistserver/fault_provider.py

```diff
@@ -4,18 +4,6 @@
 import time
 from pyworklistserver import user_config
 
-__NONASCII = 'æÆøØåÅßäöüÄÖÜ' #just an arbitrarily selected list of non ascii characters
```

Comment:
Why was this not detected before commit? Did you run the tests first?

Reply:
A mistanke by us. Will be more careful next time